### ISSUE TITLE
Contact properties not searchable

### ISSUE THREAD

**Issue by ekrahmer** [nextcloud/contacts#3143]
### Describe the bug

Typing in the content of a property (eg 85521 representing postal code) shows no matches 

### Steps to reproduce

1. insert "85521" into the search field

### Expected behavior

The NC contacts app should be able to provide search functionality for property content and, in this case, show all entries with postcode equal "85521" 

### Actual behavior

result list empty
contact displayed is not refreshed

### Contact version

4.2.3

### PHP engine version

None

### Web server

None

### Database

None

### Additional info

NC instance is a IONOS managed Nextclud. No access to CLI

**Comment by rokejulianlockhart** [comment]
Has this ever been available?

**Comment by ChristophWurst** [comment]
I don't think so. See https://github.com/nextcloud/contacts/pull/3358 for a related change. Neither are all properties used for search nor does the backend index all properties to be searchable.

**Comment by PetrVladimirov** [comment]
@ChristophWurst thanks for developing the change. Any ETA or target NC version when we can enjoy it in production?

**Comment by ekrahmer** [comment]
Unfortunately, this has never been available. 
Some of the missing properties are essential for group management issues. For some tasks I have to bypass NC and use Thunderbird in conjunction with the Cardbook extension.
This degrades NC to a filespace with hooked CardDAV server

**Comment by PetrVladimirov** [comment]
> Unfortunately, this has never been available. Some of the missing properties are essential for group management issues. For some tasks I have to bypass NC and use Thunderbird in conjunction with the Cardbook extension. This degrades NC to a filespace with hooked CardDAV server

Thank you for sharing your experience and the workaround!

**Comment by alexanderdd** [comment]
Can this issue be triaged please? (in project "contacts bug tracker" it is in "to triage")

I just had this situation: 
- in-app search: search for specific street name
- a specific contact not listed that has this street name in address field
- when using the global search, the contact is listed

Since [2021](https://github.com/nextcloud/contacts/issues/2126) people are asking for this, IMO it is a very basic feature for a contacts app, please consider developing this.
